### ISSUE TITLE
JSON mapping files does not match examples in ingest/examples

### ISSUE THREAD

**Issue by zednis** [tetherless-world/dtdi-mineral-observation-map#16]
The JSON mapping file appears to be out of sync with the example JSON inputs from ingest/examples/

Here is what is available at https://deeptime.tw.rpi.edu/es/dtdi/mineral-observation/_search?pretty

```
{
  "mineral": {
    "mineral-name-plain": "Abelsonite",
    "chemistry-elements-list": [
      "Ni",
      "C",
      "H",
      "N"
    ],
    "rruff-chemistry-html": "Ni<sup>2+</sup>C<sub>31</sub>H<sub>32</sub>N<sub>4</sub>",
    "mineral-mindat-url": "http://www.mindat.org/min-1.html",
    "year-first-published": "1975",
    "country-of-type-locality": "USA",
    "ima-number": "IMA1975-013",
    "fleischers-groupname": "\u00a0",
    "ima-chesmistry-html": "NiC<sub>31</sub>H<sub>32</sub>N<sub>4</sub>",
    "rruff-chemistry-concise": "Ni^2+^C_31_H_32_N_4_",
    "ima-chesmistry-concise": "NiC_31_H_32_N_4_"
  },
  "locality": {
    "country": "USA",
    "dated-locality-mindat-id": "180470",
    "max-age": "330",
    "lon-deg": "-111",
    "decimal-degree": "39.653611111111,-111.59833333333",
    "dated-locality-label": "Blackbird Mine (Pioneer Mine),Big Indian District (Big Indian Wash - Lisbon Valley Area),San Juan Co., Utah, USA",
    "lon-sec": "-54",
    "min-age": "13",
    "lon-min": "-35",
    "locality-mindat-url": "http://www.mindat.org/loc-14378.html",
    "locality-mindat-id": "14378",
    "region": "Utah, USA",
    "lat-min": "39",
    "locality-label": "Utah, USA",
    "lat-sec": "13",
    "dated-locality-mindat-url": "http://www.mindat.org/loc-180470.html",
    "lat-deg": "39",
    "lon": -111.59833333333333,
    "lat": 39.65361111111111
  }
}
```

and this is the JSON in the ingest/examples/bulk_pretty example JSON file:

```
{  
  "locality":[  
    {  
      "country":"Canada",
      "region":"Québec, Canada",
      "mine":"Poudrette quarry (Demix quarry; Uni-Mix quarry; Desourdy quarry; Carrière Mont Saint-Hilaire), Mont Saint-Hilaire, La Vallée-du-Richelieu RCM, Montérégie, Québec, Canada",
      "coordinates":{  
        "lat":"45.56277778",
        "lon":"-73.14166667"
      },
      "mindat":"http://www.mindat.org/loc-599.html"
    }
  ],
  "description":"Named after the Abenaki, the indigenous people who inhabited the area around Mont Saint-Hilaire in southern Québec, prior to the arrival of European settlers.",
  "mindat":"http://www.mindat.org/min-2.html",
  "mineral":{  
    "name":"Abenakiite-(Ce)",
    "rruffChemistryConcise":"Na_26_Ce^3+^_6_(SiO_3_)_6_(PO_4_)_6_(CO_3_)_6_(S^4+^O_2_)O",
    "rruffChemistryHTML":"Na<sub>26</sub>Ce<sup>3+</sup><sub>6</sub>(SiO<sub>3</sub>)<sub>6</sub>(PO<sub>4</sub>)<sub>6</sub>(CO<sub>3</sub>)<sub>6</sub>(S<sup>4+</sup>O<sub>2</sub>)O",
    "elements":"Na Ce Si O P C S"
  }
}
```

**Comment by zednis** [comment]
What does "dated-locality-mindat-url" mean?

**Comment by zednis** [comment]
I think we should revisit this JSON mapping.
- Where is mineral specimen?
- Why is min and max age under locality?  It should be under mineral specimen.
- restructure the coordinate information into an object so its not represented as 10 properties under locality
- what do the dated vs non-dated mindat IDs mean?
- change the property name "dated-locality-label"

**Comment by zhongh** [comment]
1) The updated mapping file should work with the JSON file created by ingest/example/parseOriginalData.py
I didn't upload the updated sample JSON file because it can be generated with the python script

2) So, for each locality record, it also contains a sub-record call "Dated Locality", which I am not sure of what it means. And "dated-locality-mindat-url" is the mindat url for this "dated locality" associated with the locality in this observation. @xgmachina 

3) Regarding the suggestions on JSON mapping revisiting, I can create and organize another issue for our discussion? @zednis @xgmachina

**Comment by zhonghao1014** [comment]
fixes #20

**Comment by zednis** [comment]
Resolved by #29
